db/migrate/20180512180546_add_push_notifications_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPushNotificationsToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :push_notifications, :boolean, default: true, null: false
+  end
+end
```

Comment:
I don't see any change in the `User` model.

Reply:
you gotta love Rails 😏

Reply:
I guess he meant we still need to validate the presence of `push_notifications` to get a human-readable error message?